docs/faqs.md

```diff
@@ -0,0 +1,132 @@
+---
+id: faqs
+title: Frequently Asked Questions
+hide_title: false
+---
+
+Below are some commonly asked questions that you may encounter that can assist you along the way when working with Cumulus.
+
+### General
+
+<details>
+  <summary>How do I deploy a new instance in Cumulus?</summary>
+
+  Answer: For steps on the Cumulus deployment process go to [How to Deploy Cumulus](../deployment/deployment-readme).
+</details>
+
+<details>
+  <summary>What prerequisites are needed to setup Cumulus?</summary>
+
+  Answer: You will need access to the AWS console, Cloudtamer.io, and an [Earthdata login](https://urs.earthdata.nasa.gov/) before you can deploy Cumulus.
```

Comment:
```suggestion
  Answer: You will need access to the AWS console and an [Earthdata login](https://urs.earthdata.nasa.gov/) before you can deploy Cumulus.
```
If it's outside NASA they may just have console access. For NGAP users they'd have to get it through CloudTamer.